README.md

```diff
@@ -217,6 +217,24 @@ Stop the tunnel with: `make <env> stop-tunnel`
 ## Other useful commands
 Type `make` to get the list of all available commands.
 
+## Known caveats
+
+### Fly version
+
+Current mechanism of updating fly binary only checks if the date of your local
+binary is older than one on the server. If you have been using previous fly
+version recently, it won't get updated. In that case, delete the binary in /bin
```

Comment:
that's the 1st download. You can't do fly sync without fly. Or with fly that doesn't support it. Before fly sync, curl runs and it uses the described mechanism to tell if it should download the file or not.